pennylane/tape/measure.py

```diff
@@ -76,6 +78,20 @@ def __init__(self, return_type, obs=None, wires=None, eigvals=None):
         # Queue the measurement process
         self.queue()
 
+    def __copy__(self):
+        cls = self.__class__
+        copied_op = cls.__new__(cls)
+
+        copied_op.return_type = self.return_type
```

Comment:
Why is `return_type` still hanging around for the constituent ops? (shouldn't all this logic have migrated wholy to the `MeasurementProcess`?

Reply:
This is the `MeasurementProcess` :) I'll change the variable name